curriculum/challenges/english/10-coding-interview-prep/take-home-projects/build-a-nightlife-coordination-app.md

```diff
@@ -14,7 +14,7 @@ Here are the specific user stories you should implement for this project:
 
 **User Story:** As an unauthenticated user, you can view all bars in my area.
 
-**User Story:** As an authenticated user, you can add myself to a bar to indicate you am going there tonight.
+**User Story:** As an authenticated user, you can add myself to a bar to indicate you are going there tonight.
```

Comment:
You could have fixed the whole sentence as you were here, "you can add myself" sounds horrendously

Reply:
Pushed another commit fixing these sentences.